src/main/plugins/org.dita.base/plugin.xml

```diff
@@ -172,6 +172,78 @@ See the accompanying LICENSE file for applicable license.
       <val>true</val>
       <val default="true">false</val>
     </param>
+    <param name="build-step.clean-temp" desc="Run process clean-temp" type="enum">
+      <val default="true">true</val>
+      <val>false</val>
+    </param>
+    <param name="build-step.branch-filter" desc="Run process branch-filter" type="enum">
+      <val default="true">true</val>
+      <val>false</val>
+    </param>
+    <param name="build-step.chunk" desc="Run process chunk" type="enum">
+      <val default="true">true</val>
+      <val>false</val>
+    </param>
+    <param name="build-step.clean-preprocess" desc="Run process clean-preprocess" type="enum">
```

Comment:
This is totally internal step that has no direct meaning in DITA.

It's the step that will go through all files in temp and try to rewrite the filenames to something that remotely looks like what the source was. So if you have `topic.dita` as the source file and preprocess2 will mangle this to `<HASH>.dita` in temp dir, this step will rename `<HASH>.dita` back to `topic.dita`, taking uplevels into account.

So this step is not something I would expect end users to understand without docs.

Reply:
@infotexture IIRC this step does more than just restore filenames. But I'll check.